static/app/views/settings/organizationMembers/organizationMemberDetail.tsx

```diff
@@ -353,8 +354,9 @@ class OrganizationMemberDetail extends AsyncView<Props, State> {
           </Panel>
         )}
 
-        <RoleSelect
-          enforceAllowed={false}
+        <OrganizationRoleSelect
+          enforceAllowed={hasTeamRoles}
+          deprecateAdmin={organization.features.includes('team-roles')}
```

Comment:
Looks like some extra stuff snuck in here unrelated to the padding fix? cc @leedongwei

Reply:
Yep my bad. Took it out.